src/Stratis.Features.Collateral/ConsensusRules/MandatoryCollateralMemberVotingRule.cs

```diff
@@ -65,8 +56,11 @@ public override Task RunAsync(RuleContext context)
             // Verify that the miner is including all the missing votes now.
             Transaction coinbase = context.ValidationContext.BlockToValidate.Transactions[0];
             byte[] votingDataBytes = this.votingDataEncoder.ExtractRawVotingData(coinbase);
+
+            // If there are no voting data then just return, we could be dealing with pending polls
+            // that were never executed (picked up) by other nodes.
```

Comment:
Perhaps re-phrase: "The current node could have voted when the rules were different so can't necessarily expect the miner to follow in his footsteps. TODO: Perhaps only apply the rule for one round.".